docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md

```diff
@@ -0,0 +1,31 @@
+# Strong Motion Deployment Best Practices (azimuth/orientation and miniSEED channel names)
+
+This document acts to serve as a guide for field best practices when deploying and orienting strong motion sensors and recorders in the GeoNet sensor network. For the purpose of this document both analogue strong motion FBA sensors and FBA/MEMS recorders will be referred to as sensors. All GeoNet strong motion sites currently operate with a 200Hz sampling rate therefore their miniSEED channel naming convention will always have a prefix of HN*.  
+
+## Strong motion site types 
+
+Regarding instrument installation, there are three types of strong motion site (excluding structural arrays):
+
+1. Sites in buildings
+2. Sites in GeoNet field infrastructure (cabinet, vault, VSAT hut)
+3. Sites in boreholes
+
+### 1) Sites in buildings
+
+Sensors at sites in buildings are installed inside cases which are typically aligned with the walls of the building. The orientation of these sensors should be in line with the edges of their host cases, which are aligned with the walls of the host building. In short, these sensors should be aligned with the walls of the host building. Their orientation can then be taken from the orientation of these walls. The miniSEED channel naming for these sites will be HNZ|1|2 (axial). If the building structure happens to be aligned to true north, there will be no change to this naming schema and the azimuth will be recorded as 0 (to maintain metadata consistency/continuity).
```

Comment:
suggesting some minor wording modifications below:


Sensors at sites in buildings are generally aligned with the walls of the host building. These sensors installed inside cases which are typically aligned with the walls of the building. Their orientation should be in line with the edges of their host cases, which are aligned with the walls of the host building, and can then be taken from the orientation of these walls. The miniSEED channel naming for these sites will be `HNZ|1|2` (axial). If the building structure happens to be aligned to true north, there will be no change to this naming schema and the azimuth will be recorded as `0` (to maintain metadata consistency/continuity).

Reply:
I've updated : )